api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -197,7 +203,61 @@ public void test_updateUserWithRetries_register_then_unregister() {
         .isEqualTo(TierAccessStatus.DISABLED);
   }
 
-  private DbUser registerUser(DbUser user) {
+  @Test
+  public void testUpdateUserWithRetriesForAccessRenewal() {
+    providedWorkbenchConfig.access.enableAccessRenewal = true;
+    providedWorkbenchConfig.access.enableDataUseAgreement = true;
+    providedWorkbenchConfig.accessRenewal.expiryDays = (long) 365;
+    DbUser dbUser = new DbUser();
+
+    // The user is still compliant and has bypass set
+    dbUser.setDataUseAgreementCompletionTime(
+        Timestamp.from(
+            START_INSTANT.minus(
+                (providedWorkbenchConfig.accessRenewal.expiryDays - 1), ChronoUnit.DAYS)));
+    dbUser.setDataUseAgreementSignedVersion(userService.getCurrentDuccVersion());
+    dbUser = userService.updateUserWithRetries(registerUserNow, dbUser, Agent.asUser(dbUser));
+
+    assertThat(userAccessTierDao.findAll()).hasSize(1);
+    assertThat(userAccessTierDao.getAllByUser(dbUser).get(0).getTierAccessStatusEnum())
+        .isEqualTo(TierAccessStatus.ENABLED);
+
+    // User is compliant and we remove dua bypass
+    dbUser.setDataUseAgreementCompletionTime(
```

Comment:
nit: this line seems unnecessary, the timestamp hasn't changed from above. If it's included here to make this snippet of test code atomic, I would instead suggest splitting this chunk out into a separate test method entirely